htdocs/acknowledgements/acknowledgements.php

```diff
@@ -76,4 +76,4 @@
 
 echo "</html>";
 
-?>
+
```

Comment:
Is there a reason to be keeping this extra space in all these files?

Reply:
I think the reason is that this is how PHPCBF corrects the old code.

Reply:
The main reason to remove the closing tags is because then PHP isn't sensitive to whether there's extra space/lines at the end of the file or not, so it doesn't really matter. (with the closing tags it'll send any whitespace after the tag as a literal.)